.github/workflows/e2e.yaml

```
@@ -2,13 +2,22 @@ name: End to End testing
 on: [pull_request]
 
 jobs:
-  build:
+  test-end-to-end:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --workspace --all-targets
```

While we are at it we can also remove `--all-targets` as we don't not need to build tests, etc.

```suggestion
          args: --workspace
```